nncf/onnx/quantization/quantize.py

```diff
@@ -46,6 +46,8 @@ def quantize_impl(model: onnx.ModelProto,
         raise RuntimeError('Quantization algorithm from the ONNX backend '
                            'does not support operation types in the ignored '
                            'scopes yet')
+    if target_device == TargetDevice.CPU_SPR:
```

Comment:
What happen if user uses directly `PostTrainingQuanitzation` for quantization pipeline?
Could you add the new `target_device` in `/tests/onnx/quantization/test_ptq_params.py` in `test_target_device()`?

Reply:
done